### ISSUE TITLE
Permissions incorrect when transferring from remote to local filesystem

### ISSUE THREAD

**Issue by shiruken** [imthenachoman/unraid-templates#8]
I regularly use rclone to transfer remote files (via SFTP) to my Unraid server shares. However, despite specifying the UID/GID in the template, the permissions of files transferred from a remote to the local filesystem are `root:root` rather than `nobody:users`. This results in the transferred files being inaccessible to other users or applications attempting to access the files.

The [rclone docker installation instructions](https://rclone.org/install/#install-with-docker) note this issue:

> By default, the rclone binary inside a Docker container runs with UID=0 (root). As a result, all files created in a run will have UID=0. If your config and data files reside on the host with a non-root UID:GID, you need to pass these on the container start command line.

It seems like the PUID and PGID are not having the intended effect.

**Comment by imthenachoman** [comment]
Are you doing this through the GUI from this Docker container?

**Comment by shiruken** [comment]
Yes, this is using the rclone GUI available through the docker container. My only modification during container installation is to add a host path mapped to `/data` on the container:

![image](https://user-images.githubusercontent.com/867617/183686301-44163d65-947a-45e9-b240-9984ecfe39b0.png)

I set up two configs: one remote SSH connection and one local disk. Using the Explorer, I transfer a file from remote to local.

![image](https://user-images.githubusercontent.com/867617/183688329-635d2ff8-8f23-4a07-8e9b-4f8188481198.png)

![image](https://user-images.githubusercontent.com/867617/183688389-93f6bf09-7908-4545-8502-e11218414323.png)

However, when I examine the ownership of the transferred file on Unraid, it is assigned `root:root` rather than `nobody:users`, with permissions that prevent other users or applications from modifying it.

![image](https://user-images.githubusercontent.com/867617/183688632-601a0b0b-7b3e-4f8f-987a-96e1aaa0498f.png)

Looking within the container, rlcone is running as `root` rather than the `PUID` and `PGID` specified in the template, which explains why the transferred file has the wrong ownership/permissions.

![image](https://user-images.githubusercontent.com/867617/183689451-6d37d9f7-62cb-48c2-a234-a2a007c0a8e3.png)
